include/deal.II/fe/fe.h

```diff
@@ -304,8 +304,8 @@ class FESystem;
  *
  * <h4>Interpolation matrices in two dimensions</h4>
  *
- * In addition to the fields discussed above for 1D, a constraint matrix is
- * needed to describe hanging node constraints if the finite element has
+ * In addition to the fields discussed above for 1D, a constraint matrix
```

Comment:
I guess you missed this one?

Reply:
Perhaps you meant to make this following change?

constraint matrix -> constraints object